core/auth/v2/auth_test.go

```diff
@@ -2,13 +2,14 @@ package v2_test
 
 import (
 	"crypto/sha256"
-	disperser_rpc "github.com/Layr-Labs/eigenda/api/grpc/disperser/v2"
-	"github.com/Layr-Labs/eigenda/api/hashing"
-	"github.com/Layr-Labs/eigenda/common/replay"
 	"math/big"
 	"testing"
 	"time"
 
+	disperser_rpc "github.com/Layr-Labs/eigenda/api/grpc/disperser/v2"
+	"github.com/Layr-Labs/eigenda/api/hashing"
+	"github.com/Layr-Labs/eigenda/common/replay"
+
```

Comment:
nit: remove new line

Reply:
ty! above nits and grammar errors addressed in [commit](https://github.com/Layr-Labs/eigenda/pull/1613/commits/7193742c451cd73dd126aaa42cb102f59a5420b1)